docs/source/services/timelock_service/migration.rst

```diff
@@ -30,6 +30,21 @@ migration occurred correctly.
 1. **Request Logs**: Find the TimeLock leader, and search its request logs for calls to the ``fastForwardTimestamp``
    endpoint. These should be called with a ``currentTimestamp`` query parameter, which should be the bound in the
    embedded timestamp bound store before the migration.
+
+   This bound should be noted down before an upgrade, perhaps with ``cqlsh`` or other interface to the DB:
+
+   .. code-block:: none
+
+      cqlsh> USE keyspace;
+      cqlsh:keyspace> SELECT * FROM "_timestamp";
+
+       key    | column1 | column2 | value
+      --------+---------+---------+---------------------
+       0x7473 |  0x7473 |      -1 | 0x0001020304050607
+
+   Otherwise, if using Cassandra we can attempt to determine this bound, by finding the most recent occurrence of
+   ``[CAS] Setting cached limit to {}.`` in the AtlasDB client logs (across all AtlasDB clients).
```

Comment:
Worth noting that this is logged at info by the DebugLogger (you could even link to the "logging" docs page).